web/react/components/mention_list.jsx

```diff
@@ -20,6 +20,10 @@ module.exports = React.createClass({
         PostStore.addMentionDataChangeListener(this._onChange);
         var self = this;
 
+        $('.post-right__scroll').scroll(function(){
+            self.forceUpdate();
```

Comment:
Primarily I just wanted to update the location of the mentions box whenever the user scrolls. If there's a better way to do this then let me know.